depends/Makefile

```diff
@@ -103,6 +105,10 @@ ifneq ($(qt_packages_),)
 native_packages += $(qt_native_packages)
 endif
 
+ifneq ($(zmq_packages_),)
```

Comment:
Would it work if you removed this `ifneq` and tacked `$(zmp_packages_)` on to the end of that long `packages +=` line on line 101?